src/renderer/editor/components/inspectors/script-inspector.tsx

```diff
@@ -358,6 +362,16 @@ export class ScriptInspector<T extends (Scene | Node), S extends IScriptInspecto
                         <InspectorColor object={property} property="value" label={label} step={iv.options?.step ?? 0.01} onChange={(v) => this._applyExportedValueInScenePlayer(iv.propertyKey, iv.type, v)} />
                     );
                     break;
+                case "Node":
+                    if (typeof iv.defaultValue == "string") {
+                        const defaultValue = this.editor.scene?.getNodeByName(iv.defaultValue);
```

Comment:
Identifying nodes by id is better than name as ids are unique and names are not

Reply:
Im using node name only for the specifying the default value, since the id wont be known a head of time. 
the editor would look for a mesh with the name specified by defaultValue, and if it found it, it would use the id as the property value